addons/finger/functions/fnc_perFrameEH.sqf

```diff
@@ -0,0 +1,49 @@
+/*
+ * Author: TheDrill, PabstMirror
+ * The perFrameEventHandler to draw the icons
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [fromCBA] call ace_finger_fnc_perFrameEH;
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+private["_drawColor", "_fovCorrection", "_iconSize", "_timeLeftToShow"];
+
+if (!alive ACE_player) then {GVAR(fingersHash) = HASH_CREATE;};
+// Conditions: canInteract
+if !([ACE_player, ACE_player, ["isNotInside"]] call EFUNC(common,canInteractWith)) then {GVAR(fingersHash) = HASH_CREATE;};
+//make sure player is dismounted or in a static weapon:
+if ((ACE_player != vehicle ACE_player) && {!((vehicle ACE_player) isKindOf "StaticWeapon")}) then {GVAR(fingersHash) = HASH_CREATE;};
+
+_fovCorrection = ((worldToScreen (positionCameraToWorld [1000, 0, 10000])) select 0) - 0.5;
```

Comment:
This might break if the zoom level is big enough to leave that point offscreen (5º FOV)